src/components/Table/_index.scss

```
@@ -0,0 +1,26 @@
+.table {
+    th.sortable {
+        cursor: pointer;
+
+        @each $sort in ['asc', 'desc'] {
+            &.#{$sort}::after {
+                background-image: escape-svg($table-sort-icon);
+                content: '';
+                height: 1rem;
+                margin: 0 0.5rem;
+                position: absolute;
+                width: 1rem;
+
+                @if($sort == 'desc') {
+                    transform: rotate(180deg);
+                }
+            }
+        }
```

Much more readable.
```suggestion
        .table-heading-sortable::after {
                background-image: escape-svg($table-sort-icon);
                content: '';
                height: 1rem;
                margin: 0 0.5rem;
                position: absolute;
                width: 1rem;
                
                &.table-heading-sortable-desc {
                    transform: rotate(180deg);
                }
        }
```
Not sure if the above is correct though.